metaflow/flowspec.py

```diff
@@ -397,10 +397,10 @@ def merge_artifacts(
         ----------
         inputs : Inputs
             Incoming steps to the join point.
-        exclude : List[str], optional
+        exclude : List[str], optional, default None
```

Comment:
it seems redundant but the parsing expects both optional and default to properly generate everything. Otherwise, it will just generate `Optional[List[str]]` instead of `Optional[List[str]] = None`